### DISCUSSION TITLE
.NET 8 Custom Code support for Azure Logic Apps (Standard): is it supported on Mac OS / ARM64 platform?

### DISCUSSION THREAD

**Discussion by demiliani** [Azure/logicapps#1247] [Help Needed]
I'm using Azure Logic Apps Standard extension in Visual Studio Code on Mac OS Sequoia for creating a Logic App with .NET 8 Custom code (see https://techcommunity.microsoft.com/blog/integrationsonazureblog/announcement-introducing-net-8-custom-code-support-for-azure-logic-apps-standard/4167893).
Project (Logic App + .NET Function) is created and compiled successfully. But I'm not able to open the integrated designer.
Error in VS Code is this:

<img width="1096" alt="image" src="https://github.com/user-attachments/assets/eefe6524-ed3f-4eba-861e-df2f49d4e3ed">
 It seems that the .azurelogicapps/dependencies/FuncCoreTools is always empty.
Azure Function Core Tools are successfully installed on the system.
Anyone can confirm is this works on Mac OS or on Windows on Arm or not? It works correctly on Windows 11.
Thanks in advance.

**Comment by ondfisk** [comment]
Trying to create a standard logic app workflow as described here: <https://learn.microsoft.com/en-us/azure/logic-apps/create-standard-workflows-visual-studio-code> does not work on Windows 11 on ARM either:

```bash
4:27:38 PM: Starting Design Time Api
4:27:38 PM: No connection keys found to verify
NodeJs Binaries: C:\Users\rasmusl\.azurelogicapps\dependencies\NodeJs
4:27:38 PM: Running command: "C:\Users\rasmusl\.azurelogicapps\dependencies\FuncCoreTools\func host start --port 8000" with pid: "15288"...
Downloading dependency from: https://nodejs.org/dist/v18.20.8/node-v18.20.8-win-arm64.zip
FuncCoreTools Binaries: C:\Users\rasmusl\.azurelogicapps\dependencies\FuncCoreTools

Azure Functions Core Tools
Core Tools Version:       4.1.2+9a84203d4b061d9bd9baca563e30ea796d04e115 (64-bit)
Function Runtime Version: 4.1040.300.25317

DotNetSDK Binaries: C:\Users\rasmusl\.azurelogicapps\dependencies\DotNetSDK
Local binary .NET SDK version 6.0.428
Local binary .NET SDK version 8.0.413
4:27:39 PM: Azure Logic Apps Standard Runtime Dependencies validation and installation completed successfully.
[2025-08-14T14:27:40.543Z] Error configuring services in an external startup class.
[2025-08-14T14:27:40.546Z] Error configuring services in an external startup class. System.Private.CoreLib: An attempt was made to load a program with an incorrect format. (0x8007000B).
[2025-08-14T14:27:40.548Z] A host error has occurred during startup operation 'a35635a3-1ce5-4bc4-80c1-b60f7d7f18c8'.
[2025-08-14T14:27:40.549Z] Microsoft.Azure.WebJobs.Script: Error configuring services in an external startup class. System.Private.CoreLib: An attempt was made to load a program with an incorrect format. (0x8007000B).
Value cannot be null. (Parameter 'provider')
[2025-08-14T14:27:40.566Z] Host startup operation has been canceled
```

The error message `An attempt was made to load a program with an incorrect format. (0x8007000B).` seems to indicate that the tools are x64 only.
